data/json/npcs/refugee_center/surface_refugees/NPC_Rhyzaea_Johnny.json

```diff
@@ -126,7 +126,8 @@
     "dynamic_line": "I was a counsellor actually, I worked for my band, Gitxsan.  Did a lot of mental health and addictions for people that had been through some really tough stuff.  Maybe not zombies eating your child level tough, but surprisingly not far off.  My people have gone through some real messy crap.",
     "responses": [
       { "text": "Tell me about your family.", "topic": "TALK_REFUGEE_Rhyzaea_Family" },
-      { "text": "Tell me about band.", "topic": "TALK_REFUGEE_Rhyzaea_Band" }
+      { "text": "Tell me about band.", "topic": "TALK_REFUGEE_Rhyzaea_Band" },
```

Comment:
This should say "your band".

Which is my own typo, but I'd appreciate it if you could fix it while you're in here.